--- catdoc-0.93.4/src/rtfread.c.pix	2004-12-06 10:33:25.000000000 +0100
+++ catdoc-0.93.4/src/rtfread.c	2004-12-06 10:34:02.189999040 +0100
@@ -213,7 +213,7 @@
 /* 				com.name, com.numarg); */
 				if (com.numarg == '*' && data_skip_mode == 0) {
 					data_skip_mode=group_count;
-				} else if (com.numarg == '\r') {
+				} else if (com.numarg == '\n' || com.numarg == '\r') {
 					end_paragraph(&bufptr);
 				} else if (com.numarg == '~') {
 					add_to_buffer(&bufptr,0xA0);/* NO-BREAK SPACE */
