diff --git a/src/rtfread.c b/src/rtfread.c
index 86a57fb..e15e9bb 100644
--- a/src/rtfread.c
+++ b/src/rtfread.c
@@ -219,7 +219,7 @@ int parse_rtf(FILE *f) {
 /* 				com.name, com.numarg); */
 				if (com.numarg == '*' && data_skip_mode == 0) {
 					data_skip_mode=group_count;
-				} else if (com.numarg == '\r') {
+				} else if (com.numarg == '\n' || com.numarg == '\r') {
 					end_paragraph(&bufptr);
 				} else if (com.numarg == '~') {
 					add_to_buffer(&bufptr,0xA0);/* NO-BREAK SPACE */
