diff --git a/src/charsets.c b/src/charsets.c
index e64e24c..d1cdc2f 100644
--- a/src/charsets.c
+++ b/src/charsets.c
@@ -87,8 +87,6 @@ uint16_t * read_charset(const char *filename) {
 		perror(path); 
 		return NULL;
 	}
-	if (input_buffer)
-		setvbuf(f,input_buffer,_IOFBF,FILE_BUFFER);
 	/* defaults */
 	new = calloc(sizeof(short int),256);
 	for (c=0;c<32;c++) {
diff --git a/src/substmap.c b/src/substmap.c
index fcd8ead..74f53ee 100644
--- a/src/substmap.c
+++ b/src/substmap.c
@@ -55,8 +55,6 @@ SUBSTMAP read_substmap(char* filename) {
 		free(map);
 		return NULL;
 	}
-	if (input_buffer)
-		setvbuf(f,input_buffer,_IOFBF,FILE_BUFFER);
 
 	while (!feof(f)) {
 		if (!fgets(line,1024,f)) continue;
